demisto_sdk/commands/unify/unifier.py

```diff
@@ -220,43 +226,27 @@ def insert_script_to_yml(self, script_type, yml_text, yml_data):
 
         clean_code = self.clean_python_code(script_code)
 
-        lines = ['|-']
-        lines.extend(u'    {}'.format(line) for line in clean_code.split('\n'))
-        script_code = u'\n'.join(lines)
-
-        if self.dir_name == 'Scripts':
-            if yml_data.get('script'):
-                if yml_data['script'] != '-' and yml_data['script'] != '':
-                    raise ValueError("Please change the script to be blank or a dash(-) for package {}"
-                                     .format(self.package_path))
-
-        elif self.dir_name == 'Integrations' or self.dir_name == 'Beta_Integrations':
-            if yml_data.get('script', {}).get('script'):
-                if yml_data['script']['script'] != '-' and yml_data['script']['script'] != '':
-                    raise ValueError("Please change the script to be blank or a dash(-) for package {}"
-                                     .format(self.package_path))
-        else:
-            raise ValueError('Unknown yml type for dir: {}. Expecting: Scripts/Integrations'.format(self.package_path))
+        if self.is_script_package:
+            if yml_data.get('script') not in ('', '-'):
+                raise ValueError(f'Please change the script to be blank or a dash(-) for package {self.package_path}')
 
-        yml_text = yml_text.replace("script: ''", "script: " + script_code)
-        yml_text = yml_text.replace("script: '-'", "script: " + script_code)
+            yml_unified['script'] = FoldedScalarString(clean_code)
 
-        # verify that our yml is good (loads and returns the code)
-        mod_yml_data = yaml.safe_load(yml_text)
-        if self.dir_name == 'Scripts':
-            yml_script = mod_yml_data.get('script')
         else:
-            yml_script = mod_yml_data.get('script', {}).get('script')
+            if yml_data['script'].get('script', '') not in ('', '-'):
+                raise ValueError(f'Please change the script to be blank or a dash(-) for package {self.package_path}')
 
-        assert yml_script.strip() == clean_code.strip()
+            yml_unified['script']['script'] = FoldedScalarString(clean_code)
 
-        return yml_text, script_path
+        return yml_unified, script_path
 
     def get_script_package_data(self):
+        # should be static method
         _, yml_path = get_yml_paths_in_dir(self.package_path, error_msg='')
         if not yml_path:
-            raise Exception("No yml files found in package path: {}. "
-                            "Is this really a package dir? If not remove it.".format(self.package_path))
+            raise Exception(f'No yml files found in package path: {self.package_path}. '
+                            'Is this really a package dir? If not remove it.')
```

Comment:
remove what?

Reply:
the given folder that doesn't include any yml file.
this happen when you switch between branches and leaving behind directories without any files (or just the unified file)